pkg/agent/afflag/afflag.go

```diff
@@ -0,0 +1,9 @@
+package afflag
+
+import (
+	"github.com/spiffe/spire/pkg/common/fflag"
+)
+
+var (
+	AgentFlagMap = map[fflag.Flag]bool{}
```

Comment:
I don't have super strong opinions about this, but I feel like this could be an exception to our normal agent/server/common pattern and instead defined in pkg/common/fflag/agent.go. I don't see a huge benefit to keeping it separate.

If we do keep a separate package, I think we could unexport this map and provide a `Load(fflag.RawConfig) error` method in this package that turns around and calls `fflag.Load` with the now-unexported `agentFlagMap` :shrug: .